protocols/banklessvault.eth/events.json

```diff
@@ -2,6 +2,8 @@
   {
     "title": "Community Call #1",
     "url": "https://docs.google.com/forms/d/e/1FAIpQLSespwUvYs0teDWPeFTi7HKCfHS-i_0StaKahcPHi_Zh4BZzbw/viewform",
-    "date": "2021-05-14T15:00:00.000Z"
+    "date": "2021-05-14T15:00:00.000Z",
+    "type": "call",
+    "protocol": "banklessvault"

```

Comment:
This `protocol` field seems like something we could infer at the request level (in `hub-ui`), i.e. since we'll know which protocol we're requesting events for, we can just transform/augment the response with this `protocol` field in `hub-ui` instead of having to define it here.